File: Using_Elasticsearch.Common/View/Models/Post/RequestFilterParametersMainScreen.cs
using System;
using System.Collections.Generic;
using System.Text;
using Using_Elasticsearch.Common.View.Base;

namespace Using_Elasticsearch.Common.View.MainScreen.Post
{
    public class RequestFilterParametersMainScreen : BasePaginationFilter
    {
        public string ColumnName { get; set; }
        public decimal MinValue { get; set; }
        public decimal MaxValue { get; set; }
        public List<string> Values { get; set; }
    }
}

File: ClientApp/src/app/shared/models/post/request-filter-parameters-main-screen.ts
import { BasePaginationFilter } from 'src/app/shared/models/base/base-pagination-filter';

export class RequestFilterParametersMainScreen extends BasePaginationFilter {
  public columnName: string;
  public minValue: number;
  public maxValue: number;
  public values: Array<string>;
}
